### DISCUSSION TITLE
Shrincked grouping by category

### DISCUSSION THREAD

**Discussion by ippocratis** [TechbeeAT/jtxBoard#902] [Ideas]
I like grouping by category and love things tidy. 
Is an option for the default view to be shrincked and not expanded posible?

**Comment by patrickunterwegs** [comment]
Hi @ippocratis , 
well, basically a setting could be added, but what do you think about saving the collapsed and expanded status of a group? So you could just expand and collapse whatever group and next time you open the view again, you'd see the same result. What do you think?

**Reply by ippocratis** [reply]
That actually sounds better.
More practical.
Thanks for having an eye on to this.

**Reply by patrickunterwegs** [reply]
It's done now in #903 and in a couple of hours in the Google Play Beta Channel ;-)
